docs/SMARTonFHIR.md

```diff
@@ -0,0 +1,35 @@
+[SMART on FHIR](https://docs.smarthealthit.org/) is a set of open specifications to integrate partner applications with FHIR servers and electronic medical records systems that have FHIR interfaces. 
+
+There is a tutorial on the managed service documentation that describes how to use the proxy to enable SMART on FHIR applications with the Azure API for FHIR. You can follow the [steps](https://docs.microsoft.com/azure/healthcare-apis/use-smart-on-fhir-proxy) for the open-source FHIR Server as well but need to configure a few settings in a different way first. 
+
+# Initial open-source steps
```

Comment:
>Initial open-source steps [](start = 2, length = 25)

Can we call them `Configuration changes`? I just find surprising word(s) `open-source` jumping out of nowhere.

Reply:
reworded this whole piece